packages/core/src/date/date.element.spec.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { render, html } from 'lit-html';
+import { createTestElement, waitForComponent, removeTestElement, componentIsStable } from '@clr/core/test/utils';
+import { CdsDate } from '@clr/core/date';
+import '@clr/core/date/register.js';
+
+describe('cds-date', () => {
+  let component: CdsDate;
+  let element: HTMLElement;
+
+  beforeEach(async () => {
+    element = createTestElement();
+    render(
+      html` <cds-date>
+        <label>date</label>
+        <input type="date" />
+        <cds-control-message>message text</cds-control-message>
+      </cds-date>`,
+      element
+    );
+
+    await waitForComponent('cds-date');
+
+    component = element.querySelector<CdsDate>('cds-date');
+  });
+
+  afterEach(() => {
+    removeTestElement(element);
+  });
+
+  it('should create component', async () => {
+    await componentIsStable(component);
+    expect(component).toBeTruthy();
+  });
```

Comment:
Date is internal and only visible locally running storybook. This is to test that it is consistent with the rest of the forms API.